src/Yoti.Auth/AttributeConverter.cs

```diff
@@ -49,7 +49,10 @@ public static BaseAttribute ConvertToBaseAttribute(AttrpubapiV1.Attribute attrib
                     return null;
 
                 default:
-                    return null;
+                    return new YotiAttribute<object>(
```

Comment:
Should be YotiAttribute\<string\>.
The underlying value type is a string, and will always be a string.  It is a string representation of some other type we don't recognise, I grant you, but it is a string nonetheless.